modules/crio_arch.adoc

```diff
@@ -0,0 +1,56 @@
+= Understanding CRI-O
+
+The link:http://cri-o.io[CRI-O] container engine provides a stable, more secure, and performant
+platform for running OCI (link:https://www.opencontainers.org/[Open Container Initiative])
+compatible runtimes.
+The CRI-O container engine can be used to launch containers and pods
+via OCI-compliant runtimes like runc, the default OCI runtime, or
```

Comment:
IBM style suggests using _via_ only with routing data. 
s/via/using ?

Reply:
I already have "use" in the sentence once. I changed "via" to "by engaging"